torchinductor/config.py

```diff
@@ -101,3 +103,33 @@ class triton:
     # should we stop a fusion to allow better tiling?
     tiling_prevents_pointwise_fusion = True
     tiling_prevents_reduction_fusion = True
+
+
+# create a directory containing lots of debug information
+class trace:
```

Comment:
Maybe move this up to dynamo + inductor? It would be very useful to have all the way end to end. The debug log capture is general, and the dynamo out fx_graph is the inductor in fx_graph - but for dynamo we also would be able to add:

`record_guards`
`modified_bytecode`

etc

Reply:
There are some uses of inductor alone, so I want the debugging tools to work without dynamo.  There is also a one-to-many relationship between the two, so they don't fit neatly together.